editor/src/components/inspector/flex-direction-control.tsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react'
+import { ElementInstanceMetadataMap } from '../../core/shared/element-template'
+import { ElementPath } from '../../core/shared/project-file-types'
+import { Icons, useColorTheme } from '../../uuiui'
+import { EditorDispatch } from '../editor/action-types'
+import { useEditorState, useRefEditorState } from '../editor/store/store-hook'
+import { FlexDirection } from './common/css-utils'
+import {
+  filterKeepFlexContainers,
+  metadataSelector,
+  selectedViewsSelector,
+} from './inspector-common'
+import {
+  removeFlexDirectionStrategies,
+  runStrategies,
+  updateFlexDirectionStrategies,
+} from './inspector-strategies'
+
+interface FlexDirectionToggleProps {
+  flexDirection: FlexDirection
+}
+
+export const FlexDirectionToggle = React.memo<FlexDirectionToggleProps>(({ flexDirection }) => {
+  const dispatch = useEditorState((store) => store.dispatch, 'FlexDirectionToggle dispatch')
+  const metadataRef = useRefEditorState(metadataSelector)
+  const selectedViewsRef = useRefEditorState(selectedViewsSelector)
+
+  const colorTheme = useColorTheme()
+
+  const handleColumnClick = React.useCallback(
+    (e: React.MouseEvent) =>
+      maybeSetFlexDirection(
+        dispatch,
+        metadataRef.current,
+        selectedViewsRef.current,
+        e.button === 2 ? null : 'column',
```

Comment:
Why not `e.button === 0 ? 'column' : null,`?

Reply:
no reason, I refactored it to check the left mouse button